cli/commands.go

```diff
@@ -36,7 +36,7 @@ var (
 		{
 			Name:      "activate",
 			ShortName: "a",
-			Usage:     "",
+			Usage:     "Copies the step with specified --id, and --version, into provided path. If --version flag is unset, the latest version of step will be used. If --copyyml flag is set, step.yml will copied into given path.",
```

Comment:
Copy the step with specified --id, and --version, into provided path. If --version flag is not set, the latest version of the step will be used. If --copyyml flag is set, step.yml will be copied to the given path.